### PR TITLE
feat: list user links in a table

### LINKED ISSUE #17: List links in a table using `$page.user.links`
Since https://github.com/whizzes/linx-dashboard/pull/16 capability to list links owned by the session user is possible.

Links must be listed on: `/links` page, where a user will be able to check on its existing links data.

### PR DESCRIPTION
**Pull request by Michael-Liendo**

### CODE REVIEW

File: src/routes/(app)/components/LinksList/LinksTable.svelte [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<script>
+  import { page } from '$app/stores';
+  import TableItem from './TableItem.svelte';
+  const date = new Date();
+
+  console.log($page.data.user.links);
```
Comment (LeoBorai) [inline src/routes/(app)/components/LinksList/LinksTable.svelte]: Please, remove this console log call

```suggestion
```

File: src/routes/(app)/components/LinksList/LinksTable.svelte [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+                class="p-4 text-xs font-medium tracking-wider text-left text-gray-500 uppercase dark:text-white"
+              >
+                Hash/ID
+              </th>
+              <th
+                scope="col"
+                class="p-4 text-xs font-medium tracking-wider text-left text-gray-500 uppercase dark:text-white"
+              >
+                URL
+              </th>
```
Comment (LeoBorai) [inline src/routes/(app)/components/LinksList/LinksTable.svelte:24]: I think we could have a "Created At" column now.
This PR: https://github.com/whizzes/linx/pull/33 exposes the value so we can consume it here.

**Review by LeoBorai — CHANGES_REQUESTED** [review]
Looking good! I think we could have the created at column as well and also remove the routes on "routes/(app)/links".

File: src/routes/(app)/components/LinksList/LinksTable.svelte (hunk 2 above)
Comment (Michael-Liendo) [reply src/routes/(app)/components/LinksList/LinksTable.svelte:24]: sure!

**Comment by Michael-Liendo** [comment]
> Looking good! I think we could have the created at column as well and also remove the routes on "routes/(app)/links".

You can give me more context about remove the routes?

**Comment by LeoBorai** [comment]
> > Looking good! I think we could have the created at column as well and also remove the routes on "routes/(app)/links".
> 
> You can give me more context about remove the routes?

I think we could either:

- Remove these routes here: https://github.com/whizzes/linx-dashboard/tree/main/src/routes/(app)/links
- Or use the same route (https://github.com/whizzes/linx-dashboard/tree/main/src/routes/(app)/links) to list links

**Comment by Michael-Liendo** [comment]
> > > Looking good! I think we could have the created at column as well and also remove the routes on "routes/(app)/links".
> > 
> > 
> > You can give me more context about remove the routes?
> 
> I think we could either:
> 
> * Remove these routes here: https://github.com/whizzes/linx-dashboard/tree/main/src/routes/(app)/links
> * Or use the same route (https://github.com/whizzes/linx-dashboard/tree/main/src/routes/(app)/links) to list links

Oh, yeah!

**Review by dararod — APPROVED** [review]

**Pull request merged**